### PR TITLE
Add dummy payment section

### PR DESCRIPTION
**Pull request by IKarbowiak**
Add dummy payment section with card numbers which can be used to checking different responses.

### CODE REVIEW

**Comment by IKarbowiak** [comment]
I am not sure if `Developer's Guide/Running Saleor` is a good place for that, but I can't find any better one.

File: docs/developer/available-plugins/dummy-credit-card.mdx [hunk 1]
```diff
@@ -0,0 +1,120 @@
+---
+title: Dummy Credit Card
+---
+
+## Key concepts
+
+Dummy Credit Card is a Saleor testing gateway. If you want to customize it, you can do it
```
Comment (korycins) [inline docs/developer/available-plugins/dummy-credit-card.mdx:7]: `If you want to customize it` -> Shouldn't be `to configure it` ? Not sure if the `customize` is a proper word here.

**Review by korycins — APPROVED** [review]

File: docs/developer/available-plugins/dummy-credit-card.mdx (hunk 1 above)
Comment (krzysztofwolski) [inline docs/developer/available-plugins/dummy-credit-card.mdx:7]:
```suggestion
Dummy Credit Card is a Saleor testing gateway. If you want to configure it, you can do it
```

**Comment by patrys** [comment]
Task linked: [SALEOR-522 Add documentation for new version of dummy payment](https://app.clickup.com/t/2549495/SALEOR-522)

**Review by karolkielecki — APPROVED** [review]

**Pull request merged**